desktop/src/app/components/navbar/taskbar-editor.html

```diff
@@ -0,0 +1,13 @@
+<div id="editor">
+    <button id="button" (click)="openModal()">
+        <span
+            class="mdi mdi-account-alert"
+            *ngIf="currentEditor === 'anonymous'"
+        ></span>
+        <span
+            class="mdi mdi-account-check"
+            *ngIf="currentEditor !== 'anonymous'"
+        ></span>
+        <span id="editor-initials" class="nav-link px-2">{{getInitials()}}</span>
```

Comment:
In Angular templates sollte man für die Performance Funktionsaufrufe verhindern. Siehe z.B. https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496